src/PlotRecipes.jl

```diff
@@ -19,12 +19,13 @@ end
 #     ones(nsites(sit)), sit
 # end
 
-RecipesBase.@recipe function f(var::AbstractVector, pnt::AbstractLocations)
+RecipesBase.@recipe function f(var::AbstractVector, pnt::AbstractPlaces)
```

Comment:
I still don't think this is right? AbstractLocations have location data (coordinates, grid cells, something), whereas AbstractPlaces are just places with no locality data, no?

Reply:
I think it's good like this. We need to be able to plot something for AbstractPlaces, but at the moment there's only a way to do it for AbstractLocations. Since AbstractLocations <: AbstractPlaces, for any case where there is location data, this will still work (though if it's an AbstractGrid, then getcoords() will return the grid and this recipe won't be used). For anything without locality information you have to define coordinates() yourself to say how it needs to be plotted.

Reply:
In fact AbstractLocations is precisely the thing that doesn't need a recipe, because it will always pass things off through getcoords() to its grid/point/whatever component - to which end, we should probably have an AbstractPoints as a counterpart to AbstractGrid in EcoBase, and a recipe for that - but maybe not today!

Reply:
I disagree, but since it's not causing any problems atm we can always fix it later.